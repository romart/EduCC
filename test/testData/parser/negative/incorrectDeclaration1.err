test/testData/parser/negative/incorrectDeclaration1.c:1:2: error: declaration of anonymous struct must be a definition
test/testData/parser/negative/incorrectDeclaration1.c:6:2: warning: declaration does not declare anything
test/testData/parser/negative/incorrectDeclaration1.c:6:2: error: unexpected token ELSE '(null)' instead of {
test/testData/parser/negative/incorrectDeclaration1.c:3:3: error: use of undeclared identifier 'EX'
test/testData/parser/negative/incorrectDeclaration1.c:6:3: error: expression is not assignable
test/testData/parser/negative/incorrectDeclaration1.c:8:3: error: unexpected token BAD_CHARACTER '(null)' instead of (
test/testData/parser/negative/incorrectDeclaration1.c:10:3: error: unexpected token FOR '(null)' instead of (
test/testData/parser/negative/incorrectDeclaration1.c:1:4: error: unexpected token } '(null)' instead of (
