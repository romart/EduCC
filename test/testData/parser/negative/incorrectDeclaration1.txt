FILE test/testData/parser/negative/incorrectDeclaration1.c
  ERROR TYPE