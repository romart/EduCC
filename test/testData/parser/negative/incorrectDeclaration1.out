test/testData/parser/negative/incorrectDeclaration1.c:6:2: warning: declaration does not declare anything
